content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -16,13 +17,13 @@ Services with consistent DNS names instead of IP addresses.
 
 <!-- body -->
 
-Kubernetes DNS schedules a DNS Pod and Service on the cluster, and configures
-the kubelets to tell individual containers to use the DNS Service's IP to
-resolve DNS names.
+Kubernetes publishes information about Pods and Services which is used
+to program DNS.  Kubelet configures Pods' DNS so that running containers
```

Comment:
Hi @thockin .
Does Kubernetes 'program DNS', or provide or implement DNS?

nit: At the end of a sentence after a period, use a single space.

Reply:
DNS **is** a separate concern (it's usually, but not always, CoreDNS that does the work).
Maybe “configure cluster DNS”, where cluster DNS could be italicized or even be a glossary tooltip.